api/pacman-api-compliance/src/main/java/com/tmobile/pacman/api/compliance/controller/FilterController.java

```diff
@@ -269,7 +269,21 @@ public ResponseEntity<Object> getListOfCategories(@RequestParam(name = "ag", req
         }
         return ResponseUtils.buildSucessResponse(response);
     }
-    
+
+    @GetMapping(value = "/v1/filters/issuestatus")
+    public ResponseEntity<Object> getListOfIssueStatus(@RequestParam(name = "ag", required = true) String assetGroup,
```

Comment:
Do we need this parameters? I don't see it being used anywhere

Reply:
will make required as false.